File: TransCoder_tokenized_test_set_functions/cpp/PRINT_UNIQUE_ROWS.cpp
void printArray ( int arr [ ] [ 5 ], int row, int col ) {
  unordered_set < string > uset;
  for ( int i = 0;
  i < row;
  i ++ ) {
    string s = "";
    for ( int j = 0;
    j < col;
    j ++ ) s += to_string ( arr [ i ] [ j ] );
    if ( uset . count ( s ) == 0 ) {
      uset . insert ( s );
      cout << s << endl;
    }
  }
}

File: TransCoder_tokenized_test_set_functions/java/PRINT_UNIQUE_ROWS.java
public static void printArray ( int arr [ ] [ ], int row, int col ) {
  HashSet < String > set = new HashSet < String > ( );
  for ( int i = 0;
  i < row;
  i ++ ) {
    String s = "";
    for ( int j = 0;
    j < col;
    j ++ ) s += String . valueOf ( arr [ i ] [ j ] );
    if ( ! set . contains ( s ) ) {
      set . add ( s );
      System . out . println ( s );
    }
  }
}